ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -218,12 +201,19 @@ class CommandBarButton: UIButton {
                                                     bottom: 8.0,
                                                     right: 10.0)
     }
+}
+
+// MARK: CommandBarButton UIPointerInteractionDelegate
 
-    private struct ColorConstants {
-        static let normalTintColor: UIColor = Colors.textPrimary
-        static let normalBackgroundColor = UIColor(light: Colors.gray50,
-                                                   dark: Colors.gray600)
-        static let highlightedBackgroundColor = UIColor(light: Colors.gray100,
-                                                        dark: Colors.gray900)
+extension CommandBarButton: UIPointerInteractionDelegate {
+    @available(iOS 13.4, *)
```

Comment:
@harrieshin Will we still need this line if our minimum is now 15?

Reply:
yeah this can be removed for sure. the main is already min io14 anyway.